tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -6,6 +6,13 @@ class CompletionSuite extends BaseCompletionSuite {
 
   override def requiresJdkSources: Boolean = true
 
+  override val compatProcess: Map[String, String => String] = Map(
```

Comment:
This can be removed, we don't test for 2.11

Reply:
This caused the test failure, because it overrides `compatProcess` in `BaseCompletionSuite`, now it should work